### DISCUSSION TITLE
Allow namespaced installation

### DISCUSSION THREAD

**Discussion by yonahd** [kedacore/keda#1909] [Ideas]
### Proposal

The installation of Keda should be allowed to be run with role in specific namespaces and not cluster wide ones.

### Use-Case

An example of using this set of manifests is if you run your services in a in a cluster where teams have access to specific namespaces and cannot use cluster wide permissions, This would allow a keda per namespace 

### Anything else?

This possibly would work by just using roles instead of clusterRoles (and CRB).

**Comment by tomkerkhove** [comment]
You can limit KEDA to only scale a certain namespace, but we cannot reduce the cluster roles since this is a requirement for the metrics server that we have to use.

**Comment by nadolualex** [comment]
Has this been implemented? Is it still necessary to create the `ClusterRoles`?
